### PR TITLE
CB-79, CB-80 & CB-81 Brand refresh, update cookie control and update deps

### PR DESCRIPTION
**Pull request by ediblecode**
https://nicedigital.atlassian.net/browse/CB-79
https://nicedigital.atlassian.net/browse/CB-80
https://nicedigital.atlassian.net/browse/CB-81

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
     "start": "cross-env NODE_ENV=development webpack-dev-server --hot",
     "build": "cross-env NODE_ENV=production webpack --config webpack.config.prod.js",
     "// TESTS": "",
-    "test": "jest --notify",
-    "test:coverage": "jest --notify --coverage",
+    "test": "jest --notify --detectOpenHandles",
+    "test:coverage": "jest --notify --detectOpenHandles --coverage",
```
Comment (johndavey72) [inline package.json:12]: --detectOpenHandles flag has some performance penalty apparently, but I guess it's easier than identifying the tests that aren't exiting cleanly

File: src/index.test.ts [hunk 2]
```diff
@@ -37,13 +35,9 @@ describe("Load cookie control tests", () => {
 	});
 
 	it("should CookieControl object properties from existing cookie with revoked consent", () => {
```
Comment (johndavey72) [inline src/index.test.ts:37]: missing word in the name of this test - "use/get"

File: src/index.test.ts [hunk 3]
```diff
@@ -55,13 +49,9 @@ describe("Load cookie control tests", () => {
 	});
 
 	it("should CookieControl object properties from existing cookie with consent", () => {
```
Comment (johndavey72) [inline src/index.test.ts]: missing word in name of test?

**Review by johndavey72 — APPROVED** [review]

**Review by johndavey72 — CHANGES_REQUESTED** [review]
Please update button colours as per AC3 in CB-80

File: package.json (hunk 1 above)
Comment (ediblecode) [reply package.json:12]: I prefer it not complaining when the tests finish so happy to leave it as we do elsewhere

**Comment by ediblecode** [comment]
@johndavey72 I've pushed fixes, please could you re-review?

**Review by johndavey72 — APPROVED** [review]

**Review by johndavey72 — APPROVED** [review]

**Review by johndavey72 — DISMISSED** [review]

**Review by johndavey72 — APPROVED** [review]

**Pull request merged**